scss/detail.scss

```diff
@@ -1,22 +1,67 @@
+// variables
+$container-height: 416px;
+$container-width: 900px;
+$badge-height: 119px;
+$badge-width: 95px;
+
 .container-book {
   background-color: $white-secondary;
   box-shadow: 4px 4px 10px 0 $shadow;
   display: flex;
-  height: 416px;
-  width: 900px;
+  height: $container-height;
+  width: $container-width;
+
+  @media(max-width: 1024px) {
+    flex-direction: column;
+    align-items: center;
+    height: 570px;
+    width: 336px;
+  }
 }
 
 .container-img {
+  position: relative;
   height: 100%;
   max-width: 309px;
   padding: 24px;
   width: 100%;
+  
+  @media(max-width: 1024px) {
+    height: 337px;
+    padding: 38px 48px 0 48px;
+    width: 336px;
+  }
 }
 
 .container-detail {
   height: auto;
   padding: 24px;
   width: auto;
+
+  @media(max-width: 1024px) {
+    height: 233px;
+    padding: 21px;
+    width: 336px;
+  }
+}
+
+.badge {
+  display: flex;
+  height: $badge-height;
+  width: $badge-width;
+  position: absolute;
+  transform: rotate(15deg);
+  padding: 10px;
+  right: -5px;
+  top: 0px;
+
+  @media(max-width: 1024px) {
+    height: calc($badge-height + 25%);
```

Comment:
Does this works? I thought that in order to use `calc` with sass variables it was necessary to interpolate the variable :thinking: